examples/namespace_classes.cf

```diff
@@ -1,32 +1,79 @@
-
-body common control
+bundle agent __main__
 {
-      bundlesequence => { "xxx:globalclass", "main" };
-      inputs => { "namespace_class1.cf" };
+  methods:
+      "MyNamespace:my_bundle";
+
+  reports:
+
+    a_bundle_scoped_class_in_my_namespaced_bundle::
+      "In $(this.namespace):$(this.bundle) I see 'a_bundle_scoped_class_in_my_namespaced_bundle::'";
```

Comment:
`this.namespace` seems odd at this point in the file since you haven't specified one yet with a `body file control` body. Ok. I see. Many of these will be "negative" cases. OK :+1:

Reply:
Also, I use this.namespace to show that by default your in the default namespace.